include/seqan3/core/algorithm/configuration.hpp

```diff
@@ -459,7 +459,7 @@ template <typename lhs_fn_t,
 constexpr auto operator|(lhs_fn_t && lhs_fn,
                          rhs_fn_t && rhs_fn)
 //!\cond
-    requires is_configuration_combinable_with_v<lhs_fn_t, rhs_fn_t>
+    requires is_configuration_combinable_with_v<remove_cvref_t<lhs_fn_t>, remove_cvref_t<rhs_fn_t>>
```

Comment:
You are also removing ref, although `is_configuration_combinable_with_v<` already removes ref. Maybe that would be a good place to also remove const instead of here? Also: are you sure all types of const configurations are combinable if their non-const counter-parts are? Maybe @rrahn explicitly only wanted to remove ref when comparing?

Reply:
As far as I know there should not be any difference whether a configuration is const or not.